clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -1182,12 +1174,25 @@ llvm::Value *CodeGenFunction::EmitLoadOfCountedByField(
     return nullptr;
 
   Indices.push_back(Builder.getInt32(0));
-  Res = Builder.CreateInBoundsGEP(
-      ConvertType(QualType(RD->getTypeForDecl(), 0)), Res,
-      RecIndicesTy(llvm::reverse(Indices)), "..counted_by.gep");
+  return Builder.CreateGEP(ConvertType(QualType(RD->getTypeForDecl(), 0)), Res,
```

Comment:
I think you still need `InBounds`. Is there a reason why `inbounds` is removed?

Reply:
Looks like I might have caught a previous bogus commit here when I had to create a new PR because git is ass. I'll readd it.